### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by MarkoNaumovic** [codesandbox/codesandbox-client#4339]
<h1>💥 Crash Report</h1>

  <h2>What were you trying to accomplish when the crash occurred?</h2>

  > Please use this issue template to describe what you were doing when you encountered this crash. While we are able to fill in some details automatically, it's not always enough to reproduce!

  <h3>Link to sandbox: [link]() (optional)</h3>

  <h3>Crash Details</h3>

  <details>
  <summary>Environment</summary>

  | Browser |  Version  | Operating System | CodeSandbox Version |
  | ------- | --------- | ---------------- | ------------------- |
  | firefox | 77.0.0 | Windows 10           | d77d0891b          |

  **Route:**
  https://codesandbox.io/s/marko-4phjf?file=/src/styles.scss

  </details>

  <details>
  <summary>Error Message</summary>

  ```bash
  TypeError: o is undefined
  om/<@https://codesandbox.io/static/js/app.0bf2a6caf.js:571:1031889
om@https://codesandbox.io/static/js/app.0bf2a6caf.js:571:1031417
ua@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1733563
Ka@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1741828
Es@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1780016
ml@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1764404
hl@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1764327
il@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1761655
Yo/<@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1719838
../../node_modules/react-dom/node_modules/scheduler/cjs/scheduler.production.min.js/t.unstable_runWithPriority@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1795174
qo@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1719547
Yo@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1719785
$o@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1719718
el@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1758441
Sa@https://codesandbox.io/static/js/common.92a9f0a9c.chunk.js:1:1737238
forceRerender@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1511668
track/this.callback@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1632477
flush@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1635307
flush@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1631531
C/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1520945
../../node_modules/overmind/es/index.js/o</e.prototype.emit@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1514741
createAction/</</<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1523384
i@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1525056
i/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1518192
i@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1518859
o/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1518936
w/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1525337
o@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1518670
i@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1518154
o/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1518323
E/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1525437
o@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1517924
i@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1525056
i@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1518246
o/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1518323
j/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1525194
o@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1517924
i@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1525056
i@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1518246
o/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1518323
j/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1525194
o@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1517924
i@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1525056
t@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1525094
createAction/</<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1523253
createAction/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1523232
createContext/<.actions</<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1522543
value/this.channel.onMessage@https://codesandbox.io/static/js/app.0bf2a6caf.js:571:386783
value@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1548361
value/<@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1555692
decode@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1549121
value@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1555354
value/this.conn.onmessage@https://codesandbox.io/static/js/vendors~app.f3ef99e7e.chunk.js:1:1551494

  
    in om
    in je
    in Unknown
    in div
    in elements__Container
    in Unknown
    in WithTheme(Component)
    in div
    in t
    in div
    in t
    in div
    in div
    in styled.div
    in div
    in elements__Container
    in je
    in Unknown
    in Unknown
    in Unknown
    in t
    in t
    in div
    in elements__Content
    in n
    in withRouter(n)
    in div
    in elements__Container
    in Unknown
    in withRouter()
    in t
    in Unknown
    in je
    in l
    in t
  ```

  </details>

**Comment by garethx** [comment]
@MarkoNaumovic  What were you trying to accomplish when the crash occurred?

**Comment by lbogdan** [comment]
Closing this, as it should be already fixed by #4176 .
